packages/application-generic/package.json

```diff
@@ -87,6 +87,7 @@
     "@novu/termii": "^0.13.0",
     "@novu/testing": "^0.13.0",
     "@novu/twilio": "^0.13.0",
+    "@novu/africastalking": "^0.13.0",
```

Comment:
Sorry for being annoying but let's rename the folder and packaging name to `africas-talking` due the company's branding. 🙏🏻